raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -163,6 +163,7 @@ def next_channel_from_routes(
 def try_new_route(
         channelidentifiers_to_channels: typing.ChannelMap,
         available_routes: typing.List[RouteState],
+        old_initiator_state: typing.Optional[InitiatorTransferState],
```

Comment:
I can put it as first argument, but I think that in this particular case the naming should not be `initiator_state`. It would be confusing to read as it does not have the same meaning as in the other functions.

Reply:
Ok, no strong opinion about this so if you think it's fine i am ok with it as well.